buildenv/jenkins/openjdk_tests

```diff
@@ -115,6 +115,9 @@ def PLATFORM_MAP = [
     ]
 ]
 
+if ( params.TARGET == "" ) {
```

Comment:
Any reason for not doing `if ( !params.TARGET ) {` ? By using it, we catch other invalid cases.